### PR TITLE
updated template for AutoCode

### PR DESCRIPTION
**Pull request by yudjee**

### CODE REVIEW

File: src/common/Button/Button.jsx [hunk 1]
```diff
@@ -2,6 +2,6 @@ import React from 'react';
 
 import styles from './styles.module.css';
 
-export const Button = ({buttonText, handleClick}) => (
-	<button onClick={handleClick}>{buttonText}</button>
+export const Button = ({buttonText, handleClick, 'data-testid': dataTestId}) => (
```
Comment (MillerOlesiaJob) [inline src/common/Button/Button.jsx:5]: Let's use just `dataTestId` or we can discuss it

File: src/store/slices/coursesSlice.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+	initialState,
+	reducers: {
+		// setCourses: 
+		// saveCourse: 
+		// deleteCourse: 
+		// updateCourse: 
+	},
+});
+
+export const { setCourses, saveCourse, deleteCourse, updateCourse } =
```
Comment (MillerOlesiaJob) [inline src/store/slices/coursesSlice.js:17]: I recommend adding a comment to line 15
`Please, use these actions in your components`

File: src/common/Button/Button.jsx (hunk 1 above)
Comment (yudjee) [reply src/common/Button/Button.jsx:5]: discussed
leave as is

File: src/store/slices/coursesSlice.js (hunk 2 above)
Comment (yudjee) [reply src/store/slices/coursesSlice.js:17]: added

**Review by MillerOlesiaJob — APPROVED** [review]

**Pull request merged**